pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
     "omegaconf>=2.1.1",
     "parse>=1.19.0",
     "pluggy>=1.0",
+    "pre-commit-hooks",
```

Comment:
@astrojuanlu If you feel strongly against it, I can just copy the implementation of that particular function. I added this because `pre-commit-hooks` is an extremely lightweight library (only dependency is `ruamel.yaml`, which itself is very light).

Reply:
@merelcht what do you think?

Reply:
@merelcht I don't think that's @astrojuanlu's point. I am using `pre-commit-hooks` as a runtime dependency here, because the starter requirements get sorted.

The question is whether to vendor the requirements sorting logic or leave this lightweight dependency.